databases/_upgrade_sponsorTimes_2.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN TRANSACTION;
+
+/* Add hash field */
+CREATE TABLE "sqlb_temp_table_1" (
+	"videoID"	TEXT NOT NULL,
+	"startTime"	REAL NOT NULL,
+	"endTime"	REAL NOT NULL,
+	"votes"	INTEGER NOT NULL,
+    "incorrectVotes" INTEGER NOT NULL default '1',
+	"UUID"	TEXT NOT NULL UNIQUE,
+	"userID"	TEXT NOT NULL,
+	"timeSubmitted"	INTEGER NOT NULL,
+	"views"	INTEGER NOT NULL,
+	"category"	TEXT NOT NULL DEFAULT "sponsor",
+	"shadowHidden"	INTEGER NOT NULL,
+	"hashedVideoID"	TEXT NOT NULL
```

Comment:
Maybe put this right before shadow hidden and leave shadow hidden as the last column

Reply:
I don’t think anything is SQLite depends on column order? But I’ve swapped them in https://github.com/ajayyy/SponsorBlockServer/pull/86/commits/3167c24f75bcd9c390baff39c72a47130e2d16f8.

Reply:
It just looks better when opening the database in a gui viewer. Thanks!